src/interface/report/PlayerTile.tsx

```diff
@@ -29,7 +29,12 @@ const PlayerTile = ({ player, makeUrl, config }: Props) => {
 
   useEffect(() => {
     const load = async () => {
-      if (!player.region || !player.server || !isSupportedRegion(player.region)) {
+      if (
+        player.combatant.expansion === 'wotlk' ||
```

Comment:
I think this should be turned into a utility function that checks if the given combatant's expansion is a supported retail expansion, but we can do that in a later PR.